lib/commands/doctor/checks/check-memory.js

```diff
@@ -1,16 +1,19 @@
 'use strict';
-const os = require('os');
+const sysinfo = require('systeminformation');
 const SystemError = require('../../../errors').SystemError;
 
 const MB_IN_BYTES = 1048576;
 const MIN_MEMORY = 150;
 
 function checkMemory() {
-    const availableMemory = os.freemem() / MB_IN_BYTES;
-    if (availableMemory < MIN_MEMORY) {
-        return Promise.reject(new SystemError(`Ghost recommends you have at least ${MIN_MEMORY} MB of memory available for smooth operation. It looks like you have ${parseInt(availableMemory)} MB available.`));
-    }
-    return Promise.resolve();
+    return sysinfo.mem().then((memoryInfo) => {
+        return (memoryInfo.free / MB_IN_BYTES);
```

Comment:
this should probably be `memoryInfo.available`? memoryInfo.free seems to be the same thing that os.freemem returns, whereas available returns a more holistic number that includes available buffer & cache memory.

Also, there's no need for the additional promise code here, you could just do:
```js
return sysinfo.mem().then((memoryInfo) => {
  const availableMemory = memoryInfo.available / MB_IN_BYTES;

  if (availableMemory < MIN_MEMORY) {
    return Promise.reject(); // ...
  }

  return Promise.resolve();
});

Reply:
Oops, I saw that free was set (in systeminformation source) and jumped the gun 😬